src/main/java/fr/xephi/authme/AuthMe.java

```diff
@@ -132,6 +134,12 @@ public void onEnable() {
             stopOrUnload();
             return;
         }
+        
+        // Check if NoCheatPlus is enabled and the fly bug needs to be fixed
+        if (!getServer().getPluginManager().isPluginEnabled("NoCheatPlus")
```

Comment:
At the risk of being annoying, I'd suggest to extend the PluginHookService with NoCheatPlus. That's the place where we hook into all third-party plugins right now and you'll be able to reuse listening to events in case the plugin is enabled/disabled after AuthMe has started.